src/org/zaproxy/zap/extension/fuzz/resources/Messages.properties

```diff
@@ -397,3 +397,12 @@ fuzz.httpfuzzer.processor.acsrffuzz.panel.label.showtokens = Show token requests
 
 fuzz.httpfuzzer.processor.acsrffuzz.message.name = Anti-CSRF Token Refresh
 fuzz.httpfuzzer.processor.acsrffuzz.message.error = Failed to refresh Anti-CSRF token, request failed.
+
+fuzz.httpfuzzer.processor.tagcreator.name = Tag Creator
+fuzz.httpfuzzer.processor.tagcreator.desc = Analyse response and tag it if configured Regex is matching
+fuzz.httpfuzzer.processor.tagcreator.matchbyregex.regex = Regex:
+fuzz.httpfuzzer.processor.tagcreator.matchbyregex.tag = Tag:
+fuzz.httpfuzzer.processor.tagcreator.matchbyregex.name = Match
+fuzz.httpfuzzer.processor.tagcreator.extractbyregex.regex = RegEx:
```

Comment:
The name should be normalised (either "Regex" or "RegEx", I'd go for the former though).